packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -317,7 +317,7 @@ const hasValidParticipant = (chatMessage: ChatMessageWithStatus): boolean =>
 export const messageThreadSelectorWithThread: () => MessageThreadSelector = () =>
   createSelector(
     [getUserId, getChatMessages, getLatestReadTime, getIsLargeGroup, getReadReceipts, getParticipants],
-    (userId, chatMessages, latestReadTime, isLargeGroup, readReceipts, participants) => {
+    (userId, chatMessages, latestReadTime, isLargeGroup, readReceipts = [], participants) => {
```

Comment:
We should fix getReadReceipts instead so it is fixed for every selector.

getReadReceipts return type should be `ChatMessageReadReceipt[] | undefined`

Reply:
That makes sense. Changed return type of getReadReceipts to `ChatMessageReadReceipt[] | undefined`.